src/main/resources/app.conf

```diff
@@ -77,7 +77,12 @@ app {
     base-uri = "http://localhost:8080/iam"
     base-uri = ${?IAM_BASEURI}
 
+    # The service account token to execute calls to IAM
     service-account-token = ${?IAM_SA_TOKEN}
+
+    # The maximum tolerated inactivity period after which the cached ACLs will be refreshed
+    cache-timeout = 30 seconds
```

Comment:
I think `cache-ttl`, `cache-expiry` or something similar  would be a better name for this, `cache-timeout` sounds more more like a timeout for retrieving something from cache.